web/src/main/webapp/xslt/ui-metadata/form-builder.xsl

```diff
@@ -1549,7 +1549,27 @@
   -->
   <xsl:template mode="render-for-field-for-attribute"
                 match="gn:attribute[not(@name = ('ref', 'parent', 'id', 'uuid', 'type', 'uuidref',
-    'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href'))]"
+    'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href', 'xlink:type'))]"
+                priority="4">
+    <xsl:param name="ref"/>
+    <xsl:param name="insertRef" select="''"/>
+
+    <xsl:variable name="attributeLabel" select="gn-fn-metadata:getLabel($schema, @name, $labels)"/>
+    <xsl:variable name="fieldName"
+                  select="concat(replace(@name, ':', 'COLON'), '_', $insertRef)"/>
+    <button type="button" class="btn btn-default btn-xs btn-attr btn-nil btn-xs gn-attr-{replace(@name, ':', '_')} {if ($isDisplayingAttributes = true()) then '' else 'hidden'}"
+            id="gn-attr-add-button-{$fieldName}"
+            data-gn-click-and-spin="add('{$ref}', '{@name}', '{$insertRef}', null, true)"
+            title="{$attributeLabel/description}">
+      <i class="fa fa-plus fa-fw"/>
+      <xsl:value-of select="$attributeLabel/label"/>
+    </button>
+  </xsl:template>
+
+  <!-- add attribute for link type -->
+  <xsl:template mode="render-for-field-for-attribute"
+                match="gn:attribute[not(@name = ('ref', 'parent', 'id', 'uuid', 'type', 'uuidref',
+    'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href', 'gco:nilReason'))]"
```

Comment:
This template seem matching some elements already matched in the previous template ('xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href') 

https://github.com/geonetwork/core-geonetwork/pull/5284/files#diff-dd05931fe2e9562f24fd5ced36ad129469606fad5f2aac2cdc97da357e5d99bcR1552

So the previous template doesn't apply for these elements. 

Can you confirm which of the templates should be applied to the elements: 'xlink:show', 'xlink:actuate', 'xlink:arcrole', 'xlink:role', 'xlink:title', 'xlink:href'

Reply:
I'm no expert on XSL, so I'm not sure which templates should be applied. Sorry